### PR TITLE
Add fastcache recipe

### PR DESCRIPTION
**Pull request by synapticarbors**
Addresses #2968

### CODE REVIEW

File: recipes/fastcache/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - python -c "import fastcache; fastcache.test()"
+
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/pbrady/fastcache
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (jakirkham) [inline recipes/fastcache/meta.yaml]: Please raise an issue/PR upstream to add a `MANIFEST.in` with at least the line `include LICENSE`. Also please xref that issue/PR back here. Thanks.

**Comment by jakirkham** [comment]
LGTM. One simple comment then should be good to go.

File: recipes/fastcache/meta.yaml (hunk 1 above)
Comment (synapticarbors) [inline recipes/fastcache/meta.yaml]: Added the PR at https://github.com/pbrady/fastcache/pull/45

**Comment by jakirkham** [comment]
Thanks @synapticarbors.

**Pull request merged**